runtime/tests/third_party/dummy_device.toml

```
@@ -1,95 +1,91 @@
-schema = 1
+schema = 2
 
 [device]
 name = "dummy.device.qubit"
 
-[operators]
+[operators.observables]
```

I know TOML doesn't require this, but it might be nice for visual purposes to list table levels individually:
```suggestion
[operators]

[operators.observables]
```
what do you think?

---

Fine. I think both seems ok for me.